### ISSUE TITLE
Feature request: Option to use # for header instead of underline

### ISSUE THREAD

**Issue by rljacobson** [tadashi-aikawa/markowl#2]
My markdown files use `#` for the headings, like this:

```
# title (h1)
## section1 (h2)
### subsection (h3)
text goes here
```

It would be nice to have this option in markowl.

**Comment by tadashi-aikawa** [comment]
@rljacobson 
Thank you for the feature request!

I will try to add commands such a `Draw H2 Prefix` instead of the option.
Because I use both notations as following, depending on the situation.

`Draw H2 Line`  (Already implemented)
```
header2
-------
```

`Draw H2 Prefix`
```
## header2
```

What do you think?

**Comment by rljacobson** [comment]
I agree!

**Comment by tadashi-aikawa** [comment]
@rljacobson 
I have released a new version v0.7.0.
Please see [README](https://github.com/tadashi-aikawa/markowl#draw-header-prefix) for the detail of features.

It would probably be tomorrow or day after tomorrow that you can upgrade it because JetBrains needs to approve it.

Best regards.

**Comment by tadashi-aikawa** [comment]
@rljacobson

Published! please try it 👍
https://plugins.jetbrains.com/plugin/14116-markowl
